homeassistant/components/zwave_js/discovery_data_template.py

```diff
@@ -496,6 +459,7 @@ def __post_init__(self) -> None:
         """Validate inputs."""
         for speeds in self.configuration_value_to_speeds.values():
             assert len(speeds) > 0
+            assert sorted(speeds) == speeds
```

Comment:
Add a comment to explain that this is just for developers to not make a mistake. It's not for production purposes.

Reply:
Done.  Thanks!